bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -904,6 +904,14 @@ spec:
           - jobs
           verbs:
           - '*'
+        - apiGroups:
```

Comment:
@jopit m not sure if this change is required as following mail of Auth and API , doesn't mentions